models/model.js

```diff
@@ -67,3 +68,14 @@ exports.getArticleCommentsModel = (id) => {
       return { comments: rows };
     });
 };
+
+exports.postCommentModel = (id, data) => {
+  const itemsInsertStr = pg(
```

Comment:
pg seems like overkill here, you just have 3 values to insert so I'd go with $1, $2, $3 rather than using a library